app/controllers/concerns/v2/pagination.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module V2
+  # Default options and configuration for pagination
+  module Pagination
+    extend ActiveSupport::Concern
+
+    ParamType = ActionController::Parameters
+
+    included do
+      def pagination_limit
+        permitted_params[:limit] || 10
+      end
+
+      def pagination_offset
+        permitted_params[:offset] || 0
+      end
+
+      def page
+        pagination_offset + 1
```

Comment:
This is definitely not good, the relationship between offset and page should be a multiplication of the page number with the per_page limit.